devops/scripts/fetchOldTags.sh

```diff
@@ -37,8 +37,8 @@ done
 
 # Validation of required arguments and env values
 if [ -z "$LICENSE_PLATE" ] || [ -z "$ENV" ] || [ -z "$APP_NAME" ]; then
-  echo "License plate, environment, and app name are required."
   echo "Usage: $0 --license_plate=VALUE --env=VALUE --app_name=VALUE [--prefix=VALUE] [--min_tags=VALUE]"
+  echo "License plate, environment, and app name are required."
```

Comment:
In the same file below can we please convert the MIN_TAGS to number like
from 
`
  | if $MIN_TAGS > 0 then .[:-$MIN_TAGS] else . end
  | .[]
  `
To 
`
| if $MIN_TAGS > 0 then .[:-($MIN_TAGS | tonumber)] else . end
  | .[]
`

Reply:
The explicit conversion is not required as jq can handle the value and our run identifiers are always numeric.  Great question!